api/pkg/transformer/executor/transformer.go

```diff
@@ -84,7 +83,7 @@ func (st *standardTransformer) Execute(ctx context.Context, requestBody types.JS
 		st.logger.Debug("preprocess response", zap.Any("preprocess_response", preprocessOut))
 	}
 
-	reqBody, err := json.Marshal(preprocessOut)
+	reqBody, err := preprocessOut.AsOutput()
```

Comment:
Do we need to do `.AsOutput()` for the post processing?

Reply:
for the executor we don't need that, because the postprocessing result won't be directly sent through network, the result will be use as a field in the response object